cypress/integration/login.js

```diff
@@ -7,18 +7,33 @@ describe('Login functionality', function() {
         cy.get('.Select.not-valid').should('be.visible')
     })
 
-    it('Should be able to login with role User', function () {
+    // it('Should be able to login with role User', function () {
```

Comment:
Hey there, everything looks great, except for the comments - usually we try to leave comments out of source control, that is we do not push them. When developing stuff locally they're great, but in the source control they're just boilerplate text. Let's try and remove this commented code :)